proto/buf/alpha/registry/v1alpha1/user.proto

```diff
@@ -53,6 +54,19 @@ message OrganizationUser {
   OrganizationRoleSource organization_role_source = 4;
 }
 
+message UserSelectedPlugin {
+  // owner of the plugin
+  string owner = 1;
+  // name of the plugin
+  string name = 2;
+}
+
+message UserPluginPreference {
```

Comment:
What is this? needs docs at the minimum

Reply:
That is this used for? The documentation just says what the fields are, I'm not sure what this structure is for, and how it fits into the Buf product.